README.md

```diff
@@ -214,9 +234,11 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
 | <a href="https://www.manypixels.co/gallery"><img width='700' src='https://user-images.githubusercontent.com/74038190/184810217-55c81d5a-321f-401d-aa27-4864c8a197a6.png' alt="Manypixels_free_illustrations" /></a> | [Many Pixels](https://www.manypixels.co/gallery)  <br><br> ✔ Free <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• PNG<br>• SVG<br>• AI<br><br> ✌️Author <br> Getillustrations                             |
 | <a href="https://www.pixsellz.io/material-icons-library"><img width='700' src='https://user-images.githubusercontent.com/92253071/193480907-db472e82-2421-4476-9f3f-df0db0667333.png' alt="Material_Icons_free_Library" /></a> | [Material Icons](https://www.pixsellz.io/material-icons-library)  <br><br> ✔ Free  <br><br>Format <br> • Figma<br> • Sketch<br><br> ✌️Author <br> Pixsellz                                                                 |
 | <a href="https://github.com/MariaLetta/mega-doodles-pack/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184917535-88553471-26eb-4bf2-a532-b719341a66c8.png' alt="Mega_Doodles_Pack_free" /></a> | [Mega Doodles Pack](https://github.com/MariaLetta/mega-doodles-pack/)  <br><br> ✔ Free <br><br> |
+| <a href="https://www.drawkit.com/product/mental-health-icons"><img width='700' src='https://user-images.githubusercontent.com/34191321/194963389-3399dedb-e379-462e-890b-bbe449b8d9da.png' alt="Mental_Health_icons_free_illustrations" /></a> | [Mental Health Icons](https://www.drawkit.com/product/mental-health-icons)  <br><br> ✔ Free  <br><br>Format <br> • SVG<br> • PNG<br> • JPG<br><br> ✌️Author <br> DrawKit                                                 |
```

Comment:
- change the src link to `https://user-images.githubusercontent.com/74038190/195569982-89003c4d-1116-4aeb-bcb9-8d0b33dbd186.png`